client/my-sites/plans-features-main/hooks/use-generate-action-callback.ts

```diff
@@ -164,6 +167,7 @@ function useGenerateActionCallback( {
 	const handleUpgradeClick = useUpgradeHandler( { siteSlug, coupon, cartHandler } );
 	const handleDowngradeClick = useDowngradeHandler( {
 		siteSlug,
+		siteUrl: siteUrl || '',
```

Comment:
WDYT if we default to `siteSlug`? 🤔

Reply:
It won't make a difference, these are either both available or not. The `getSiteSlug` [uses the site URL internally.](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/sites/selectors/get-site-slug.js#L26)